src/index.js

```diff
@@ -23,39 +23,29 @@ bot.on('ready', () => {
   console.log(`Logged in as ${bot.user.tag}!`);
 });
 
-// Testing objects for enemy and player
-let enemy;
-
-let player = {
-  name: 'Hero',
-  hp: 100,
-  atk: 20,
-  def: 7,
+const ctx = {
+  player: {
+    name: 'Hero',
+    hp: 100,
+    atk: 50,
```

Comment:
Our hero has been working out! The poor Discord bot had to deal with me spamming with a measly 20 attack :(